File: Example/SimpalmSpinnerObjc/SimpalmSpinnerObjc/ViewController.m
//
//  ViewController.m
//  SimpalmSpinnerObjc
//
//  Created by Simpalm on 17/05/19.
//  Copyright © 2019 simplam. All rights reserved.
//

#import "ViewController.h"
@import SimpalmSpinner;


@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    
    SimpalmSpinner *loader = [SimpalmSpinner new];
   // [loader showLoaderMessageWithInteractionWithMessage:@"Great Loader!" onView:self.view interaction:YES];
    [loader showLoaderWithMessageWithMessage:@"Great Loader!" onView:self.view];
   // [loader showLoaderOnView:self.view];
    loader.themeColor = [UIColor whiteColor];
    loader.spinnerBackgroundColor = [UIColor blueColor];
    loader.screenBackgroundColor = [UIColor colorWithRed:227.0/255 green:27.0/255 blue:27.0/255 alpha:0.5];
   
}


@end

File: Example/SpinnerSwift/SpinnerSwift/ViewController.swift
//
//  ViewController.swift
//  SpinnerSwift
//
//  Created by Simpalm on 17/05/19.
//  Copyright © 2019 simplam. All rights reserved.
//

import UIKit
import SimpalmSpinner

class ViewController: UIViewController {
    
    let spin = SimpalmSpinner()
    
    override func viewDidLoad() {
        
        super.viewDidLoad()
        
        self.show(i :56.0)
        
        //spin.showLoaderWithMessage(message: "Apple provide best product......!", onView: self.view)
        
        DispatchQueue.main.asyncAfter(deadline: .now() + .seconds(6), execute: {
            self.spin.dismiss()
        })
        
       spin.spinnerBackgroundColor = UIColor(red: 227.0/255, green: 27.0/255, blue: 27.0/255, alpha: 0.5)
        
       spin.themeColor = .white
        
       spin.screenBackgroundColor = UIColor(red: 227.0/255, green: 27.0/255, blue: 27.0/255, alpha: 0.5)
        
    }
    
    func show(i: Int) {
        
        spin.showLoaderWithMessage(message: "Apple provide best product......!INT", onView: self.view)
        
    }
    
    func show(i: Float) {
        
        spin.showLoaderWithMessage(message: "Apple provide best product......!FLO", onView: self.view)
        
    }
    
}


